spring-kafka/src/test/java/org/springframework/kafka/listener/AbstractConsumerSeekAwareTests.java

```diff
@@ -0,0 +1,132 @@
+package org.springframework.kafka.listener;
```

Comment:
The class has checkstyle violations and missing copyright section, author tag etc.

Reply:
Thanks! I fixed it. [a724510](https://github.com/spring-projects/spring-kafka/pull/3318/commits/a724510a8d86588d73863ac7ebe3867c30a60af7)